### ISSUE TITLE
[Bug] There ARE 1 new articles

### ISSUE THREAD

**Issue by AbelLykens** [marienfressinaud/FreshRSS#6470]
### Describe the bug

![image](https://github.com/FreshRSS/FreshRSS/assets/47113175/f9900267-4fc5-445d-974b-d6a9a8ec8398)

_Are_ should be _is_ in these notifications. Love Freshrss so thanks!!

### To Reproduce

Get 1 new unread article, have notifications enabled.

### Expected behavior

Are should be is

### FreshRSS version

 1.23.1

### Environment information

- Database version: Mariadb
- PHP version: PHP
- Installation type: From source
-Web server type: Apache
- Device: PC
- OS: Hosted on Debian 12, viewing on Win11
- Browser: Firefox 126

**Comment by math-GH** [comment]
I would say that it is worth to fix ;)

But keep in mind that different languages have different issues with numbers and grammar.

Here an example from Russian language: 2,3,4 have another grammar case then 6 upwards. (it repeats with 22,23,24, 32,33,34,42,43 and so on)

```
Есть 1 новая статья

Есть 2 новые статьи
Есть 3 новые статьи
Есть 4 новые статьи

Есть 5 новых статей
Есть 6 новых статей
```

**Comment by Manitosh-Paul** [comment]
Hello team, I am a new contributor and want to contribute to open source.
This issue can be found in file: https://github.com/FreshRSS/FreshRSS/blob/5b197b8130efed7953f42cde0cbd2217b7fb0dbe/app/i18n/el/gen.php#L132C5-L132C22

However, there is one more issue with this bug i.e.,
If there is only one unread article the message should be like this: "There is %%d new article to read on FreshRSS." otherwise, the message should be: "There are %%d new articles to read on FreshRSS."

As, I am new contibutor in PHP too, I think we can use conditional statement something like this in line number 132:
'body_new_articles' => ('%%d'==1 ? 'There is %%d new article to read on FreshRSS.' : 'There are %%d new articles to read on FreshRSS.'),

**Comment by Alkarex** [comment]
@Manitosh-Paul Thanks for trying to help :-)
As @math-GH is writing just above, the grammar of numbers is more complicated than just having two cases for singular and plural (although it would probably cover several languages well enough). So the first step is to list a couple of approaches, for instance by finding out how it is done in other projects (maybe WordPress for instance, although I have not sure they are facing this problem).

**Comment by Manitosh-Paul** [comment]
@Alkarex In that case this bug should remain unchanged due to language dynamics, so, should we focus on this bug ?

**Comment by Alkarex** [comment]
Yes, it would be nice if someone could dig a bit into that issue at some point, but it is not urgent

**Comment by Frenzie** [comment]
Wordpress uses gettext (although also some JSON thing for JS).

If you're curious about how gettext goes about things I wrote a custom implementation: http://koreader.rocks/doc/modules/gettext.html#ngettext

**Comment by aledeg** [comment]
I think we should dig how it is handled in the Symfony framework. See https://symfony.com/doc/8.0/reference/formats/message_format.html.
We might find something interesting.

**Comment by Frenzie** [comment]
It's conceptually identical to GetText.

**Comment by Alkarex** [comment]
Since https://github.com/FreshRSS/FreshRSS/pull/8670 we have a better management of plurals, so now it should be quite easy to fix this issue. PR welcome

**Comment by AbelLykens** [comment]
Nice @Frenzie !
